server/sonar-server/src/main/java/org/sonar/server/es/EsUtils.java

```diff
@@ -79,4 +82,17 @@ public static String formatDateTime(@Nullable Date date) {
     }
     return null;
   }
+
+  public static void setShards(NewIndex index, Settings settings) {
```

Comment:
Good idea. It's a better alternative than EsUtils. What I don't want is to repeat code in several places.